api/handler/acl.go

```diff
@@ -153,7 +153,10 @@ func (h *handler) bearerTokenIssuerKey(ctx context.Context) (*keys.PublicKey, er
 		return nil, err
 	}
 
-	key, err := keys.NewPublicKeyFromBytes(box.Gate.BearerToken.Signature().Key(), elliptic.P256())
+	var btoken v2acl.BearerToken
+	box.Gate.BearerToken.WriteToV2(&btoken)
+
+	key, err := keys.NewPublicKeyFromBytes(btoken.GetSignature().GetKey(), elliptic.P256())
```

Comment:
Create an issue in SDK with the link to this code. Probably we can have both `btoken.Issuer()` and `btoken.IssuerKey()` methods.